openwisp_notifications/templates/admin/openwisp_users/user/change_form_object_tools.html

```diff
@@ -0,0 +1,12 @@
+{% extends "admin/change_form_object_tools.html" %}
+
+{% load i18n admin_urls %}
+
+{% block object-tools-items %}
+  {% if request.user.is_staff and original.is_staff %}
```

Comment:
We need a test to ensure that this button only appears on User page of staff users. You don't need to a selenium test, you can achieve this with Django test. Use the `self.assertContains` to check the anchor element is present.